### ISSUE TITLE
RMG-tests still errors on eg1

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#1227]
```
Traceback (most recent call last):
 File "/home/travis/build/ReactionMechanismGenerator/RMG-tests/code/benchmark/RMG-Py/scripts/checkModels.py", line 288, in <module>
 main()
 File "/home/travis/build/ReactionMechanismGenerator/RMG-tests/code/benchmark/RMG-Py/scripts/checkModels.py", line 80, in main
 check(name, benchChemkin, benchSpeciesDict, testChemkin, testSpeciesDict)
 File "/home/travis/build/ReactionMechanismGenerator/RMG-tests/code/benchmark/RMG-Py/scripts/checkModels.py", line 92, in check
 execute(benchChemkin, benchSpeciesDict, benchThermo, testChemkin, testSpeciesDict, testThermo, **kwargs)
 File "/home/travis/build/ReactionMechanismGenerator/RMG-tests/code/benchmark/RMG-Py/rmgpy/tools/diff_models.py", line 313, in execute
 model1.species, model1.reactions = loadChemkinFile(chemkin1, speciesDict1, thermoPath = thermo1)
 File "/home/travis/build/ReactionMechanismGenerator/RMG-tests/code/benchmark/RMG-Py/rmgpy/chemkin.py", line 884, in loadChemkinFile
 reactionList = readReactionsBlock(f, speciesDict, readComments = readComments)
 File "/home/travis/build/ReactionMechanismGenerator/RMG-tests/code/benchmark/RMG-Py/rmgpy/chemkin.py", line 1307, in readReactionsBlock
 raise e
rmgpy.exceptions.ChemkinError: Unexpected species identifier C_methyl;C_rad/H2/Cs encountered in flux pairs for reaction C3H7(15) + CH3(2) <=> CCCC(11).
```

Correctly splitting the kinetics comment at `family:` in 10f1d267a11bbee726cec4950f7c3ffe9a4f1ed7 did not fully address the issue.

**Comment by mliu49** [comment]
Example of problematic reaction comment:
```
! Reaction index: Chemkin #174; RMG #1055
! Template reaction: Disproportionation
! Flux pairs: O2(2), HO2(10); C4H5(257), C4H4(332); 
! O2b;Cds/H2_d_Crad from training reaction 2
! Exact match found for rate rule [O2b;Cds/H2_d_Crad]
! Euclidian distance = 0
! Multiplied by reaction path degeneracy 4.0
! family: Disproportionation
C4H5(257)+O2(2)=C4H4(332)+HO2(10)                   4.160e+16 -1.260    3.310    
```

**Comment by mliu49** [comment]
This has technically been fixed by #1229, although there are still similar issues which are addressed by #1230.
